controllers/pkg/reconcilers/approval/reconciler.go

```diff
@@ -80,6 +81,7 @@ func (r *reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, c i
 
 // reconciler reconciles a NetworkInstance object
 type reconciler struct {
+	apiReader       client.Reader
```

Comment:
Can you provide additional info on introducing the `apiReader`? Thanks

Reply:
Added additional info to the initial comment above.